src/main/java/lotto/WinningNumbers.java

```diff
@@ -0,0 +1,29 @@
+package lotto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class WinningNumbers {
+
+    private final List<Integer> numbers;
```

Comment:
당첨번호도 구입한 로또와 제약사항이 동일하니 LottoTicket 타입을 활용하면 어떨까요?
```suggestion
    private final LottoTicket numbers;
```

Reply:
input에서 파싱을 해도 크게 문제는 없겠지만, 분리가 필요하다고 판단되시면 controller에서 파싱하면 어떨까요?
input이 얼마나 많은 책임을 수행하느냐 보다 도메인의 책임으로 적합한가가 더 우선되어야 한다고 생각합니다.
MVC 패턴을 적용해 도메인은 핵심 비즈니스 로직에만 집중할 수 있는 구조를 고민해 보시면 좋을 것 같아요!